docs/changelog.rst

```diff
@@ -12,6 +12,10 @@ Bug Fixes:
 - Fixed an issue where the ``python`` plugin would fail to build if the part
   had no Python scripts.
 
+Documentation:
+
+- Update Rust plugin doc with recent changes to the Rust toolchain.
```

Comment:
In the future, it would be a good idea to use the document name, and even a link to it.

Reply:
Added!